### ISSUE TITLE
Station Icons

### ISSUE THREAD

**Issue by vinteo** [rianadon/opensprinkler-card#4]
Wanted to open this for discussion. I have custom icons for my stations but it is not shown as this card uses hardcoded icons. My suggestion is to respect the station icons set on the station entities and use the colour to indicate whether it is running or not, kinda like how light entities work (not sure if possible).

Note the default station has different icons for running/idle but I don't think its possible to use custom icons for two different states through the entity configuration

**Comment by rianadon** [comment]
Yup icon customization is a thing I should eventually get to. And while I'm at it, I should probably support the `state_color` option as well, like the light entities.

I like having the water icons as the default because I think they're a lot clearer in what they represent than the station icons, and by default all the icons in Home Assistant are monochrome so the filled/outline/strikethrough variants fit in well.

Maybe the solution is to have a set of `disabled_icon`, `active_icon`, and `idle_icon` options defaulting to what they are currently, but when they are set to null, fall back to the station icons?

**Comment by woobenjamin** [comment]
+1
In fact, it would be awesome to combine the picture entities card with opensprinkler-card. I have used the picture-entities card previously, and put the mid:sprinkler-variant icon reflecting the general area for each station. But this is probably a little too much to ask for :-)
